### PR TITLE
Adds service / rake task for processing from moab storage.

### LINKED ISSUE #71: Allow reprocessing of preserved objects
This will allow us to update technical metadata for items that are incorrect, or don't have properties we need.

- [ ] Mount preservation roots read-only
- [ ] Need a mechanism for finding the file paths of the files in preservation given a druid. (using moab versioning gem).

### PR DESCRIPTION
**Pull request by justinlittman**
closes #71

## Why was this change made?
To allow regenerating technical metadata for items located on preservation storage.

## Was the usage documentation (e.g. openapi.yml, README, DevOpsDocs) updated?
Yes.

## Does this change affect how this application integrates with other services?

If so, please confirm:
- [ ] change was tested on stage    and/or
- [ ] test added to sul-dlss/infrastructure-integration-test

### CODE REVIEW

File: config/settings/development.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+storage_root_map:
+  default:
+    fixture_sr1: 'spec/fixtures/storage_root01'
+#    fixture_sr2: 'spec/fixtures/storage_root02'
```
Comment (jcoyne) [inline config/settings/development.yml]: cruft?

File: config/settings/test.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+storage_root_map:
+  default:
+    fixture_sr1: 'spec/fixtures/storage_root01'
+#    fixture_sr2: 'spec/fixtures/storage_root02'
```
Comment (jcoyne) [inline config/settings/test.yml]: cruft?

File: spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
```
Comment (jcoyne) [inline spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml]: why do we need these files?

Comment (justinlittman) [reply spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml]: Not strictly necessary. But the idea was to have a complete moab so as not to have to understand how the moab versioning gem works (to know which files are and are not necessary) and to be the same as pres cat fixtures.

File: config/settings/development.yml (hunk 1 above)
Comment (justinlittman) [reply config/settings/development.yml]: Removed.

File: config/settings/test.yml (hunk 2 above)
Comment (justinlittman) [reply config/settings/test.yml]: Removed.

File: spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml (hunk 3 above)
Comment (jcoyne) [reply spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml]: Is that documented somewhere?

Comment (justinlittman) [reply spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml]: It's a test fixture. Where / how would you suggest documenting?

Comment (justinlittman) [reply spec/fixtures/storage_root01/sdr2objects/bj/102/hs/9687/bj102hs9687/v0001/data/metadata/contentMetadata.xml]: README added.

**Pull request merged**